src/plugins/help/help.js

```diff
@@ -1,11 +1,6 @@
 /**
  * Help popup plugin
  *
- * Example:
- *
- *     <!-- Show a help popup at start, or if user presses "H" -->
- *     <div id="impress-help"></div>
- *
```

Comment:
This needs to stay. Plugins that display something on the screen are required to be optional. One way to make them optional is that the presentation author must explicitly add an empty div for them to appear.

In the case of the help plugin, note that it is shown automatically when the presentation is loaded. Due to this, the div must stay.